src/org/labkey/test/tests/AdminConsoleTest.java

```diff
@@ -295,51 +284,10 @@ public void testAdminConsoleCredits()
         assertTextPresent("JAR Files Distributed with the API Module");
     }
 
-
-
-    @Override
-    public List<String> getAssociatedModules()
-    {
-        return Arrays.asList("admin");
-    }
-
-    @Override
-    public void checkQueries()
-    {
-
-    }
-
-    @Override
-    public void checkViews()
-    {
-
-    }
-
     @BeforeClass
     public static void doSetup() throws Exception
     {
-        AdminConsoleTest initTest = (AdminConsoleTest)getCurrentTest();
-        initTest.createTestUser();
-    }
-
-    @Override
-    protected void doCleanup(boolean afterTest) throws TestTimeoutException
-    {
-        _userHelper.deleteUsers(false, APP_ADMIN_USER);
-    }
-
-    private void createTestUser()
-    {
-        int userId = _userHelper.createUser(APP_ADMIN_USER, true, false).getUserId();
-        setInitialPassword(userId);
-
-        ApiPermissionsHelper apiPermissionsHelper = new ApiPermissionsHelper(this);
-        apiPermissionsHelper.addMemberToRole(APP_ADMIN_USER, "Application Admin", PermissionsHelper.MemberType.user, "/");
-    }
-
-    @Override
-    protected BrowserType bestBrowser()
-    {
-        return BrowserType.CHROME;
+        AdminConsoleTest initTest = getCurrentTest();
+        initTest.createTestUsers();
```

Comment:
This `@BeforeClass` isn't necessary. `AbstractAdminConsoleTest.doSetup` would create the test users.
_I think JUnit will only run this one though, since they have the same method name._

Reply:
Confirmed and removed.